crates/virtio-queue/src/queue.rs

```diff
@@ -533,10 +556,14 @@ mod tests {
             i += 1;
             q.disable_notification().unwrap();
 
-            while let Some(_chain) = q.iter().unwrap().next() {
+            while let Some(chain) = q.iter().unwrap().next() {
                 // Here the device would consume entries from the available ring, add an entry in
```

Comment:
Can you also update the comments in this test as they are not accurate anymore? Thanks!